e2e/images/build.sh

```diff
@@ -15,16 +15,13 @@ then
     for IMAGE_NAME in $(find * -name Dockerfile -exec dirname {} \; | tr '/' '-')
     do
         docker image push -a ghcr.io/kedacore/tests-$IMAGE_NAME
-        docker image push -a docker.io/kedacore/tests-$IMAGE_NAME
     done
 else
     for IMAGE in $(find * -name Dockerfile)
     do
         IMAGE_NAME=$(dirname $IMAGE | tr '/' '-')
         pushd $(dirname $IMAGE)
-        docker build -t docker.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG -t docker.io/kedacore/tests-$IMAGE_NAME:latest .
-        docker tag docker.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG ghcr.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG
-        docker tag docker.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG ghcr.io/kedacore/tests-$IMAGE_NAME:latest
+        docker build -t ghcr.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG -t ghcr.io/kedacore/tests-$IMAGE_NAME:latest .
```

Comment:
So we are not going to produce individual tags?

Reply:
This change pushes the same tags than before or am I missing it?

Reply:
yes, we are going to push the same tags to ghcr.io but we are not going to push anything to dockerhub.
Look that there are 2 tags specified in build command:
```
ghcr.io/kedacore/tests-$IMAGE_NAME:$IMAGE_TAG
ghcr.io/kedacore/tests-$IMAGE_NAME:latest
```